src/app/routes/Router.tsx

```diff
@@ -69,34 +78,40 @@ const VerifyModelId = () => {
     return <Outlet />
 }
 
-const sectionsNoEditRoute = new Set<Section>([SECTIONS_MAP.locale])
```

Comment:
Yeah this is currently intentional. The idea is that we will create these routes "manually", since they don't share the same functionality as those that map to a schema .